test/pages/api/xAPI/statements/index.spec.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  illegalStatementCollection,
+  singleStatement,
+  statementCollection,
+} from "../../../fixtures.json";
+import dbClient from "../../../../../lib/client";
+import handlers from "../../../../../pages/api/xAPI/statements/index";
+import { testServer } from "../../../../server";
+const usr = "test-user";
```

Comment:
A new line between `import` and `const` could be _visually_ useful for devs looking at the file for the first time or after a long time.